index.html

```diff
@@ -128,13 +128,27 @@ <h2>Síguenos</h2>
             <i class="fab fa-youtube-square"></i>
           </div>
         </article>
```

Comment:
Ojo con el BEM, aquí lo correcto hubiera sido: 
follow-us --> BLOQUE 
follow-us__header --> ELEMENTO 
follow-us__icons --> ELEMENTO 
* Recordad la regla que una clase modificadora núnca va sin la clase del elemento primero, si lo véis así veréis que no os hace falta la modificadora en este caso.